plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/create-variable-product-block-editor.spec.js

```diff
@@ -119,26 +118,27 @@ test.describe( 'Variations tab', () => {
 
 				await page.waitForLoadState( 'domcontentloaded' );
 
-				await page.locator( 'text=Create "Size"' ).click();
-
-				const attributeColumn = page.getByPlaceholder(
-					'Search or create attribute'
-				);
+				await page
+					.locator(
+						'input[aria-describedby^="components-form-token-suggestions-howto-combobox-control"]'
+					)
+					.fill( attributesData.name );
 
-				await expect( attributeColumn ).toHaveValue( 'Size' );
+				await page.locator( 'text=Create "Size"' ).click();
 
 				for ( const option of attributesData.options ) {
-					await page
-						.locator(
-							'.woocommerce-new-attribute-modal__table-attribute-value-column .woocommerce-experimental-select-control__input'
-						)
-						.fill( option );
+					const container = page.locator(
+						'td.woocommerce-new-attribute-modal__table-attribute-value-column'
+					);
 
-					await page.locator( `text=Create "${ option }"` ).click();
+					const inputLocator = container.locator(
+						'input[id^="components-form-token-input-"]'
+					);
 
-					await expect(
-						page.locator( '.woocommerce-attribute-term-field' )
-					).toContainText( option );
+					await inputLocator.fill( option );
+					await page.waitForTimeout( 500 ); // needs some time to be able to press Enter.
```

Comment:
As a follow-up, it would be good to find a way to avoid using this `waitForTimeout`.